es/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Chuleta/Acordeón de Deep Learning
```

Comment:
I can't think of a good translation for "cheatsheet", as far as I know, each region has different names for this (I am from Colombia). Being the literal translation "Hoja de trampa/engaño", Could it be translated as "Hoja de ayuda" or something similar?